code/modules/mob/mob.dm

```diff
@@ -359,9 +360,14 @@
 	var/list/hearers = get_hearers_in_view(hearing_distance, src)
 	if(self_message)
 		hearers -= src
+
+	for(var/obj/item/taperecorder/rec in hearers)
+		rec.hear_raw(message, src)
```

Comment:
I'm not sure this is the most appropriate way to handle this - it's rather snowflakey. 

I'd think the *correct* way to go about this is to replace the proc call to `show_message` to a call to `Hear`, and to refactor `Hear` - Right now it's designed to only handle spoken messages, so we could probably split it into two procs (`Hear` and `Hear_non_spoken`). 

This would give us a lot more control over handling what atoms / mobs ACTUALLY hear.

Reply:
As an aside, the way you have it structured now you'll get a lot of janky chat messages

`[00:34] The defibrillator paddles <span class='notice'>The defibrillator paddles beeps: Unit is recharged.</span>`

So if nothing else you'll have to fix that - an alternative I propose is *not* including all audible messages and *just* handling emotes and gunshots specifically